hubble-be/src/main/java/com/baidu/hugegraph/service/load/LoadTaskService.java

```diff
@@ -284,6 +290,7 @@ public void updateLoadTaskProgress() {
 
     private LoadTask buildLoadTask(GraphConnection connection,
                                    FileMapping fileMapping) {
+        connection = sslService.configSSL(this.config, connection);
```

Comment:
`sslService.configSSL` should called at outside, this method should just generate LoadOptions  and LoadOptions